src/main/java/com/google/step/datamanager/RestaurantManagerDatastore.java

```diff
@@ -41,6 +42,19 @@ public Restaurant createRestaurant(String name, String photoBlobkey) {
     return new Restaurant(id, name, photoBlobkey);
   }
 
+  @Override
+  public DefaultRestaurant createDefaultRestaurant(String name, String photoUrl) {
+    Entity entity = new Entity("Restaurant");
+    entity.setProperty("name", name);
+    entity.setProperty("photoReference", photoUrl);
```

Comment:
Do we really need 2 different fields `photoReference` and `photoBlobKey`? Aren't they both just image urls?

Reply:
Because the restaurants generated from the script uses photos from google maps api, those photos must be retrieved from an url containing the photo reference. Other restaurants posted by user uses uploaded photos so they need a blob key, but photo reference and blob key need different urls.